OsmIntegrator/appsettings.json

```diff
@@ -18,7 +18,7 @@
   },
   "AllowedHosts": "*",
   "OsmCronInterval": "0 0 2 * * *",
-  "OsmExportMinDelayMins": "5",
+  "OsmExportMinDelayMins": "0",
```

Comment:
Lepiej to dać do `appsettings.Development.json`, a w `appsettings.json` zostawić 5 minut.

Reply:
fixed